docs_guidelines/index.rst

```diff
@@ -17,10 +17,10 @@ This site hosts the guidelines and examples for writing and building documentati
    :hidden:
 
    Overview <self>
-   Getting Started <getting_started>
-   Tutorials <tutorials/index>
-   How-to Guides <how_to/index>
-   API Reference <apidocs/index>
-   Explanations <explanations/index>
-   Release Notes <release_notes>
+   getting_started
+   tutorials/index
+   how_to/index
+   apidocs/index
+   explanation/index
+   release_notes
```

Comment:
The code is cleaner this way! Nice detail.

Not sure about changing "explanations" to "explanation". I saw explanations as countable in the same way as tutorials, that is, each file is "an explanation".

However, now that I check the [Diataxis page](https://diataxis.fr/explanation/), explanation is used as uncountable there.

The word "explanation" can be used as both countable and uncountable so I guess it's a matter of choosing one and sticking to it.